CHANGELOG.md

```diff
@@ -22,6 +23,32 @@ This release depends on, and has been tested with, the following Mapbox dependen
 - Mapbox Android Core `v5.0.1`
 - Mapbox Android Telemetry `v8.1.1`
 
+## Mapbox Navigation SDK 2.6.0-beta.1 - June 2, 2022
```

Comment:
Is it valid to have changelog for v2.6.0-beta.1 in v2.5 branch?

Reply:
Sure, that doesn't really matter as entries have dates and version names attached to them.

Reply:
Historically we were not including these entries though 🤷‍♂️